yarn-project/circuit-types/src/prover_coordination/epoch_proof_quote_payload.ts

```diff
@@ -84,6 +84,11 @@ export class EpochProofQuotePayload {
     };
   }
 
+  getSize(): number {
+    // 32 bytes for epochToProve, 32 bytes for validUntilSlot, 32 bytes for bondAmount, 20 bytes for prover, 4 bytes for basisPointFee
+    return 32 + 32 + 32 + EthAddress.SIZE_IN_BYTES + 4;
```

Comment:
This kind of stuff scares me for when things get out of sync. It doesn't feel like this should be called often- can we just call `toBuffer` and get length?

Reply:
Could cache the result if performance was a concern.

Reply:
perf was the concern, ill play with that idea though